packages/react-storage/src/components/StorageManager/StorageManager.tsx

```diff
@@ -25,6 +26,7 @@ function StorageManagerBase(
   {
     acceptedFileTypes = [],
     accessLevel,
+    autoUpload = true,
```

Comment:
Optional prop with default set to true, which is the current behavior (no braking change)